spec/controllers/videos_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe VideosController do
+
+  describe 'GET #show' do
+    context 'for authenticated users' do
+      before(:each) do
+        user = Fabricate(:user)
+        self.controller.login_user user
+      end
+      let(:video) { video = Fabricate(:video) }
+
+      it 'assigns @video variable' do
+        get :show, id: video.id
+        assigns(:video).should eq video
```

Comment:
I would recommend to stick with `expect().to` syntax and prefer it over `.should` as it is the newer way RSpec goes. Also to be consistent with your other specs.